src/runtime/entitlements-manager-test.js

```diff
@@ -885,6 +894,28 @@ describes.realWin('EntitlementsManager', {}, (env) => {
       await manager.consumeMeter_(ents);
     });
 
+    it('should not send pingback with invalid GAA params', async () => {
+      nowStub.reset();
```

Comment:
I don't think this line is needed

Reply:
You're right! Removing all calls to reset.